src/compiler/types.ts

```diff
@@ -3798,7 +3809,9 @@ namespace ts {
         ContainsYield = 1 << 24,
         ContainsHoistedDeclarationOrCompletion = 1 << 25,
 
-        HasComputedFlags = 1 << 29, // Transform flags have been computed.

```

Comment:
Please leave this as `1 << 29`. It is the maximum bit we can set before we outgrow the size of a v8 small integer (SMI) on an x86 system, and is a good reminder of how much room we have left. If you could, please add a comment to indicate this.